cs/configuring.texy

```diff
@@ -107,6 +108,28 @@ $configurator->addParameters([
 ]);
 \--
 
+Lze definovat i dynamické parametry, které nezpůsobí přegenerování containeru.
+
+/--php
+$configurator->addDynamicParameters([
+	'rootDir' => 'foobar',
+]);
+\--
+
+
+Přidání služeb
+--------------
+
+Služby můžeme do DI containeru přidat i ručně.
```

Comment:
Otázka je, proč bychom to měli dělat… Asi bych to popsal na situaci, kdy potřebujeme do kontaineru přidat svou instanci a doplnil bych to s popisem volby `dynamic: yes`.